### PR TITLE
Better chart for "Restarts"

### PR DESCRIPTION
**Pull request by adambraimbridge**
image

### CODE REVIEW

**Comment by adambraimbridge** [comment]
cc @commuterjoy

**Comment by commuterjoy** [comment]
+1

File: views/misc.html [hunk 1]
```diff
@@ -1,7 +1,12 @@
 
-<h2>Application restarts (deployments)</h2>
+<h2>Application restarts</h2>
+
+	<div
+		data-target='legendValue(aliasByNode(drawAsInfinite(sortByTotal(groupByNode(heroku.{apps}.*.express.start,1,"sumSeries"))),0),"total")'
```
Comment (commuterjoy) [inline views/misc.html]: try http://graphite.readthedocs.org/en/latest/functions.html#graphite.render.functions.currentAbove ?

**Comment by adambraimbridge** [comment]
Filtered out non-restarting apps

image

**Pull request merged**